e2e/multi-app-project/multi_app_project_test.go

```diff
@@ -146,32 +146,19 @@ var _ = Describe("Multiple App Project", func() {
 					AppName:     appName,
 				})
 				Expect(appShowErr).NotTo(HaveOccurred())
-				Expect(len(app.Routes)).To(Equal(1))
 
-				// Call each environment's endpoint and ensure it returns a 200
-				route := app.Routes[0]
-				Expect(route.Environment).To(Equal("test"))
-				// Since the front end was added first, it should have no suffix.
+				Expect(len(app.ServiceDiscoveries)).To(Equal(1))
```

Comment:
Can you add a *third* app so we can still test multiple services sharing one ALB?

Reply:
Sure!

Reply:
Done! added a "www" app